easybuild/easyconfigs/m/MIGRATE/MIGRATE-3.4.4-ictce-5.4.0.eb

```diff
@@ -0,0 +1,45 @@
+# 
```

Comment:
has this easyconfig a special easyblock? Else an `easyblock = ` line should be added.